### PR TITLE
rgw: Fix reshard log so we can know why the reshard is returning error

### PR DESCRIPTION
**Pull request by ofriedma**
Fix reshard log so we can know why the reshard is returning error
Signed-off-by: Or Friedmann <ofriedma@redhat.com>

### CODE REVIEW

**Comment by ofriedma** [comment]
jenkins test make check arm64

**Comment by cbodley** [comment]
@ofriedma can you please rebase?

**Comment by ofriedma** [comment]
@cbodley ping

File: src/rgw/rgw_reshard.cc [hunk 1]
```diff
@@ -869,11 +869,12 @@ int RGWReshard::list(int logshard_num, string& marker, uint32_t max, std::list<c
   int ret = cls_rgw_reshard_list(store->getRados()->reshard_pool_ctx, logshard_oid, marker, max, entries, is_truncated);
 
   if (ret < 0) {
+    lderr(store->ctx()) << "ERROR: failed to list reshard log entries, oid=" << logshard_oid << " " 
+    << "marker=" << marker << " " << cpp_strerror(ret) << dendl;
```
Comment (ivancich) [inline src/rgw/rgw_reshard.cc:873]: I think you want `cpp_strerror(-ret)`.

Comment (ivancich) [reply src/rgw/rgw_reshard.cc:873]: Never mind.

Comment (ofriedma) [reply src/rgw/rgw_reshard.cc:873]: @ivancich can you please change it to approved so we can merge it?

Thanks

**Pull request merged**